README.md

```diff
@@ -22,109 +22,88 @@
  
  OpenFOAM instance with log file hosted by http server http://www.openfoam.com/ .
  
+ #### Download the plugin binary:
+
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-openfoam/releases) page.
+
  #### Compile plugin
```

Comment:
Please correct headers/sections in this readme, see on the web page how it looks now.